ConnectSDK/lint.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<lint>
+    <issue id="InvalidPackage">
+        <ignore regexp="java.awt"/>
```

Comment:
https://github.com/zendesk/zendesk_java_common/pull/85/files

Pretty sure we don't need that anymore